src/blog/2024/04/how-to-build-an-application-with-node-red-dashboard-2.md

```diff
@@ -0,0 +1,225 @@
+---
+title: How to Build An Application With Node-RED Dashboard 2.0
+subtitle: A step-by-step guide to building a personalized, secure, and fully functional application with Dashboard 2.0.
+description: Learn to build custom applications effortlessly with Node-RED Dashboard 2.0. This step-by-step guide walks you through building a personalized, secure, and fully functional application.
+date: 2024-04-11
+authors: ["sumit-shinde"]
+image: /blog/2024/04/images/build-application-dashboard-2.png
+tags:
+    - posts
+    - node-red
+    - dashboard
+---
+
+We started developing Dashboard 2.0 a few months ago to replace the outdated Node-RED Dashboard 1.0. Initially, our goal was to replicate the old features and achieve feature parity. During the development Dashboard 2.0 has surpassed the deprecated Node-RED Dashboard, and it's now possible to build advanced applications. In this guide we'll build a Todo application to guide you through the features and explain how you can build rich and dynamic applications too.
+
+<!--more-->
+
+
+If you're new to Dashboard 2.0, refer to our blog post [Getting Started with Dashboard 2.0](/blog/2024/03/dashboard-getting-started/) to install and get things started.
+
+## Installing Flowfuse user addon
```

Comment:
@joepavitt I think in FlowFuse it should come preinstalled somehow.

Reply:
FYA @MarianRaphael @hardillb